src/components/ScenarioParameters/components/ScenarioParametersTabs/ScenarioParametersTabs.js

```diff
@@ -20,19 +21,30 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-function _buildScenarioTabList(tabs, userRoles, t) {
+function _buildScenarioTabList(tabs, userRoles, t, errors) {
   const tabListComponent = [];
+  const errorsByTab = Object.assign(
+    {},
+    ...tabs.map((tab) => ({
+      [tab.id]: tab.parameters.map((param) => param.id).filter((parameter) => Object.keys(errors).includes(parameter)),
+    }))
+  );
```

Comment:
Can we immediately store the number of errors for each tab in this loop (instead of accessing `.length` below), or 0 if there are no errors ?
Maybe the construction of `errorsByTab` could be moved to a distinct function too, its code seems to be complex enough that we might want to write unit tests on it :slightly_smiling_face: